README.md

```diff
@@ -1,4 +1,4 @@
-<img src="https://raw.githubusercontent.com/dfki-ric/pytransform3d/main/doc/source/_static/logo.png" />
```

Comment:
This is intentional so that the image is linked correctly in the long description for PyPI. I don't have a better solution so far. Could you revert this change?